config/env.js

```diff
@@ -1,21 +1,20 @@
-'use strict';
```

Comment:
I am not sure if changes on env.js is intense .

Reply:
i think it was caused by eslint --fix